homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -83,9 +81,6 @@ async def async_step_init(self, user_input=None):
                     vol.Optional(
                         CONF_AVOID, default=self.config_entry.options.get(CONF_AVOID)
                     ): vol.In([None, *AVOID]),
-                    vol.Optional(
-                        CONF_UNITS, default=self.config_entry.options[CONF_UNITS]
-                    ): vol.In(UNITS),
```

Comment:
AFAICT, the units can be selected by the user, but in the end they are not used by the integration.

Reply:
It's actually used in the update method

Reply:
Line 187/191

Reply:
Should be resolved in 9cb383f
I have adjusted the PR description accordingly